probe/endpoint/reporter.go

```diff
@@ -71,34 +71,34 @@ func (r *Reporter) Report() (report.Report, error) {
 
 func (r *Reporter) addConnection(rpt *report.Report, c *procspy.Connection) {
 	var (
-		scopedLocal  = report.MakeAddressNodeID(r.hostID, c.LocalAddress.String())
-		scopedRemote = report.MakeAddressNodeID(r.hostID, c.RemoteAddress.String())
-		key          = report.MakeAdjacencyID(scopedLocal)
-		edgeKey      = report.MakeEdgeID(scopedLocal, scopedRemote)
+		localAddressNodeID  = report.MakeAddressNodeID(r.hostID, c.LocalAddress.String())
```

Comment:
Looks like these renamings are orthogonal to the meat of this change and could be put in separately.